ocs_ci/ocs/cluster_load.py

```diff
@@ -193,89 +195,99 @@ def reach_cluster_load_percentage(self):
         # Once determined, these pods will be deleted. Then, new FIO DC pods will be
         # created, with a smaller value of 'rate' param. This in order to be more
         # accurate with reaching the target percentage
-        rate = '250M'
-        while not limit_reached:
-            self.increase_load_and_print_data(rate=rate)
+        while True:
+            wait = False if len(self.dc_objs) <= 1 else True
+            self.increase_load_and_print_data(rate='250M', wait=wait)
             if self.current_iops > self.previous_iops:
                 cluster_limit = self.current_iops
 
             latency = self.calc_trim_metric_mean(metric=constants.LATENCY_QUERY) * 1000
             latency_vals.append(latency)
             logger.info(f"Latency values: {latency_vals}")
 
-            if len(latency_vals) > 1 and latency > 250:
-                # Checking for an exponential growth
-                if latency > latency_vals[0] * 2 ** 7:
-                    logger.info("Latency exponential growth was detected")
-                    limit_reached = True
+            iops_diff = (self.current_iops / self.previous_iops * 100) - 100
+            low_diff_counter += 1 if -15 < iops_diff < 10 else 0
 
-            # In case the latency is greater than 3 seconds,
+            cluster_used_space = get_percent_used_capacity()
+
+            if len(latency_vals) > 1 and latency > 200:
+                # Checking for an exponential growth
+                if latency > latency_vals[0] * 2 ** 7 or (
```

Comment:
Please add more details about this calculation

Reply:
I will add the details as part of https://github.com/red-hat-storage/ocs-ci/pull/2493

Reply:
https://github.com/red-hat-storage/ocs-ci/pull/2493/files#diff-198938419d5d1b9b803f1d40b020c0abR214-R220